daemon/src/db.rs

```diff
@@ -1123,6 +1125,8 @@ async fn load_cfd_events(
     })
     .collect::<Result<Vec<_>>>()?;
 
+    events.sort_unstable_by(CfdEvent::chronologically);
```

Comment:
Why don't you use sql for the ordering?

Reply:
Because I want to ensure that I do the sorting _after_ I've used rayon to deserialize the events in parallel. I think rayon should preserve the order but as mentioned in the commit message, I think it is better to be explicit about this!

Reply:
Ah, got it.